backend/app/api/routes/users.py

```
@@ -0,0 +1,151 @@
+        
+        return response.data[0]
+    
+    except Exception as e:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=str(e)
+        )
+
+@router.post("/me/avatar")
+async def upload_avatar(
+    file: UploadFile = File(...),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)